umpost/conversion_driver_esm1p5.py

```diff
@@ -188,25 +252,47 @@ def convert_esm1p5_output_dir(esm1p5_output_dir):
         atm_dir_contents, fields_file_name_pattern
     )
 
-    if len(atm_dir_fields_files) > 0:
-         # Run the conversion
-        convert_fields_file_list(atm_dir_fields_files, nc_write_dir)
-    else:
+    if len(atm_dir_fields_files) == 0:
         warnings.warn(
             f"No files matching pattern '{fields_file_name_pattern}' "
             f"found in {current_atm_output_dir.resolve()}. No files will be "
             "converted to NetCDF."
         )
 
-   
+        # TODO: Check a better way of doing this
+        return  [], [] # Don't try to run the conversion
```

Comment:
This is fine as the reporting funcs receive empty sequences & do nothing with the correct type.

Reply:
Also, if `None, None` was returned, the workflow has to handle both `None` and sequence inputs. This way is simpler :-)